README.md

```diff
@@ -10,33 +10,19 @@ Un planificador semanal de comidas.
 El programa tiene como objetivo generar un planning de comidas para una persona
 o grupo de personas, adecuándose a sus necesidades.
 
+## Comprobar el código
+Para comprobar que el código del proyecto es correcto ejecutar:
+```sh
+just check
+```
+
 ## Estado del proyecto
 #### Pre-pre-Alpha.
 
 * Se han definido historias de usuario e hitos de desarrollo.
 * Se ha comenzado a implementar un modelo del problema.
 
-## Problema a solucionar
-La elección de las comidas para una persona o un grupo de personas.
-
-### Cómo pretende solucionarlo
-Partiendo de unos requisitos establecidos por el usuario, genera un planning
-de comidas semanal y una serie de informaciones asociadas.
-
-### Lógica de negocio
-Existen dos lógicas de negocio no excluyentes:
-
-#### Lógica A
-El cliente es el usuario que genera su planning semanal, pagando una
-mensualidad por el servicio. En este caso el programa habrá de ofrecer
-suficientes funcionalidades para justificar su coste.
-Posibles funcionalidades:
- - Elaboración de una lista de la compra.
- - Posibilidad de compra online en determinados supermercados.
- - Historico de comidas.
-
-#### Lógica B
-El cliente son entidades interesadas en estudios acerca de los hábitos
-alimenticios de determinadas poblaciones. Pudiendo ser supermercados,
-entidades públicas o cualquier organización interesada en el estudio de las
-preferencias alimentarias y de compra de una población.
+## Más información
+
+Puedes encontrar más información acerca de la lógica de negocio de este
+proyecto en [lógica de negocio](/docs/lógica-de-negocio.md).
```

Comment:
Por favor, no cambiéis cosas que no correspondan al objetivo.

Reply:
Okay! :+1: